utils/manifests/getComponents.go

```diff
@@ -49,7 +49,10 @@ func GetCrdsFromHelm(url string) ([]string, error) {
 			if err == io.EOF {
 				break
 			}
-			return nil, err
+			errStr := err.Error()
+			//The first line is "yaml unmarshall error:".
+			errStr = strings.TrimPrefix(errStr, strings.SplitN(errStr, "\n", 2)[0]+"\n")
```

Comment:
Why this is needed?

Reply:
When the error is returned yaml decoder
it has a line ```yaml: unmarshal errors:
  line 6921: mapping key "apiVersion" already defined at line 6427
  line 6922: mapping key "kind" already defined at line 6428
  line 6923: mapping key "metadata" already defined at line 6429
  line 6928: mapping key "spec" already defined at line 6436
  line 7723: mapping key "status" already defined at line 6914```
  we dont need the yaml:unmarshal errors: because we defined our own  meshkit error we only require the line number(s).

Reply:
Wil you add a comment so that it's clear in future what this piece of code is doing?

Reply:
Now I think @Jougan-0 does it affect anything? Having that line is no harm. 
Instead, it clears confusion IMO for eg: if error is `line 4545 duplicate field xcv` if we don't have the link it is confusing from where the error is originating is it from validation/marshal or where?
Also, this string manipulation is not worth the benefit.

Reply:
Including the model name is helpful, that's ok.

Reply:
so will you remove this logic and only keep adding pkg name in error

Reply:
Yes.